extensions/public-rest-api/src/main/java/org/eclipse/dataspaceconnector/demo/api/rest/IonDemoApiController.java

```diff
@@ -14,10 +14,13 @@
 import org.eclipse.dataspaceconnector.catalog.spi.QueryResponse;
 import org.eclipse.dataspaceconnector.catalog.spi.model.FederatedCatalogCacheQuery;
 import org.eclipse.dataspaceconnector.common.collection.CollectionUtil;
+import org.eclipse.dataspaceconnector.spi.contract.negotiation.ConsumerContractNegotiationManager;
+import org.eclipse.dataspaceconnector.spi.contract.negotiation.response.NegotiationResult;
```

Comment:
Maybe not in this PR but would be good to rename this class as it s a bit confusing.